modules/tinymce/src/core/main/ts/api/util/VK.ts

```diff
@@ -22,9 +23,11 @@ interface VK {
   PAGE_DOWN: number;
   END: number;
   HOME: number;
+  CTRL: number;
+  META: number;
 
-  modifierPressed: (e: KeyboardLikeEvent) => boolean;
-  metaKeyPressed: (e: KeyboardLikeEvent) => boolean;
+  modifierPressed: (e: KeyboardLikeEvent ) => boolean;
```

Comment:
Why is there a extra space on these?

Reply:
Just forgotten, I thought eslint would automatically remove them.
Done!